### PR TITLE
Updated proc_creation_win_iis_service_account_password_dumped.yml

### PR DESCRIPTION
**Pull request by blueteam0ps**
Updated proc_creation_win_iis_service_account_password_dumped.yml to include additional detection logic based on latest intel.

### CODE REVIEW

File: rules/windows/process_creation/win_sus_auditpol_usage.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+title: Suspicious Auditpol Usage 
+id: 0a13e132-651d-11eb-ae93-0242ac130002
```
Comment (nasbench) [inline rules/windows/process_creation/win_sus_auditpol_usage.yaml:2]: You can delete this file as this rule is already in the public repo

File: rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml [hunk 2]
```diff
@@ -20,11 +21,17 @@ detection:
         CommandLine|contains:
             - ' /list '
             - ' list '
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml:23]:
```suggestion
        CommandLine|contains: 'list '
```

File: rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml [hunk 3]
```diff
@@ -4,8 +4,9 @@ status: experimental
 description: Detects the Internet Information Services (IIS) command-line tool, AppCmd, being used to list passwords
 references:
     - https://www.elastic.co/guide/en/security/current/microsoft-iis-service-account-password-dumped.html
-author: Tim Rauch
-date: 2022/09/28
+    - https://twitter.com/0gtweet/status/1588815661085917186?cxt=HHwWhIDUyaDbzYwsAAAA
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml:7]:
```suggestion
    - https://twitter.com/0gtweet/status/1588815661085917186?cxt=HHwWhIDUyaDbzYwsAAAA
    - https://www.netspi.com/blog/technical/network-penetration-testing/decrypting-iis-passwords-to-break-out-of-the-dmz-part-2/
```

File: rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml [hunk 4]
```diff
@@ -20,11 +21,17 @@ detection:
...
         CommandLine|contains|all:
             - ' /text'
             - ' password'
-    condition: all of selection*
+    selection_cmd2:
+        CommandLine|contains:
+            - ' /@t '
+            - ' /text '
+            - ' /config '
+            - ' /@c'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml:33]: I suggest combining both selection_cmd as it's not necessary to separate them anymore.

```yml
selection_cmd:
    CommandLine|contains:
        - ' /@t' # Covers both "/@text:*" and "/@t:*"
        - ' /text'
        - ' /show'
        - ' /config'
```

File: rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml [hunk 5]
```diff
@@ -20,11 +21,17 @@ detection:
...
+    condition: 1 of selection_cmd* and selection_list
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml:34]:
```suggestion
    condition: all of selection_*
```

File: rules/windows/process_creation/win_sus_auditpol_usage.yaml (hunk 1 above)
Comment (frack113) [reply rules/windows/process_creation/win_sus_auditpol_usage.yaml:2]: https://github.com/SigmaHQ/sigma/blob/1f8e37351e7c5d89ce7808391edaef34bd8db6c0/rules/windows/process_creation/proc_creation_win_sus_auditpol_usage.yml

**Review by blueteam0ps — COMMENTED** [review]
Deleted win_sus_auditpol_usage.yaml

**Review by nasbench — APPROVED** [review]

**Review by frack113 — APPROVED** [review]

**Comment by frack113** [comment]
```yaml
    selection_base_1:
        - Image|endswith: '\appcmd.exe'
        - OriginalFilename: 'appcmd.exe'
    selection_base_1:
        CommandLine|contains: 'list '
    selection_cmd1:
        CommandLine|contains|all:
            - ' /text'
            - ' password'
    selection_cmd2:
        CommandLine|contains:
            - ' /@t '
            - ' /text '
            - ' /config '
            - ' /@c'
    condition: all of selection_base* and 1 of selection_cmd*
```

**Comment by nasbench** [comment]
> ```yaml
>     selection_base_1:
>         - Image|endswith: '\appcmd.exe'
>         - OriginalFilename: 'appcmd.exe'
>     selection_base_1:
>         CommandLine|contains: 'list '
>     selection_cmd1:
>         CommandLine|contains|all:
>             - ' /text'
>             - ' password'
>     selection_cmd2:
>         CommandLine|contains:
>             - ' /@t '
>             - ' /text '
>             - ' /config '
>             - ' /@c'
>     condition: all of selection_base* and 1 of selection_cmd*
> ```

Thanks for noticing frack. I fixed the issue by applying my original suggestion.

```yml
selection_cmd:
    CommandLine|contains:
        - ' /@t' # Covers both "/@text:*" and "/@t:*"
        - ' /text'
        - ' /show'
        - ' /config'
```

We don't need 2 `selection_cmd` since this one covers both. As the `/password` in the original one is not needed.

**Review by nasbench — APPROVED** [review]

**Review by frack113 — APPROVED** [review]

**Comment by phantinuss** [comment]
@nasbench : After the changes the "password" keyword is no longer needed for matching. This generates FPs. I would like to add it back, if there is no objection. I just found the FP. If there are cases where the "password" keyword is not needed, this might change things.

**Comment by nasbench** [comment]
> @nasbench : After the changes the "password" keyword is no longer needed for matching. This generates FPs. I would like to add it back, if there is no objection. I just found the FP and have no more detailed knowledge about IIS. If there are cases where the "password" keyword is not needed, this might change things.

Thanks for the ping. As discussed internally the FP was fixed in #3793

**Pull request merged**